src/main/java/org/opensearch/index/codec/customcodecs/Lucene95CustomCodec.java

```diff
@@ -73,40 +71,14 @@ public Set<String> getAliases() {
     private final StoredFieldsFormat storedFieldsFormat;
 
     /**
-     * Creates a new compression codec with the default compression level.
+     * Creates a new compression codec.
      *
      * @param mode The compression codec (ZSTD or ZSTDNODICT).
      */
-    public Lucene95CustomCodec(Mode mode) {
-        this(mode, DEFAULT_COMPRESSION_LEVEL);
-    }
 
-    /**
-     * Creates a new compression codec with the given compression level. We use
-     * lowercase letters when registering the codec so that we remain consistent with
-     * the other compression codecs: default, lucene_default, and best_compression.
-     *
-     * @param mode The compression codec (ZSTD or ZSTDNODICT).
-     * @param compressionLevel The compression level.
-     */
-    public Lucene95CustomCodec(Mode mode, int compressionLevel) {
+    public Lucene95CustomCodec(Mode mode) {
```

Comment:
Could we move 9.5 related codecs to `backward_codecs` ?

Reply:
@reta have moved these to a new ```backward_codecs``` directory.